modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -64,6 +64,27 @@ The following Unicode Characters are removed in identifiers:
 '\u202D', '\u202E', '\u2060', '\u2061', '\u2062', '\u2063', '\u2064', '\u2066',
 '\u2067', '\u2068', '\u2069', '\u206A', '\u206B', '\u206C', '\u206D', '\u206E',
 '\u206F', '\u2E2F', '\uFEFF', '\uFFF9', '\uFFFA', '\uFFFB'
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+MATCH (n)-[r:REL]->(m) SET n = r;
+MATCH (n)-[r:REL]->(m) SET n += r;
+MATCH (n)-[r:REL]->(m) SET r = n;
+MATCH (n)-[r:REL]->(m) SET r += n;
+----
+a|
+Using a `NODE` or `RELATIONSHIP`, instead of a `MAP`, on the RHS of a `SET` properties clause is no longer supported.
+Use the xref:functions/scalar.adoc#functions-properties[properties()] function instead to get the map of properties of nodes/relationships that
+can then be used in the `SET` clause:
```

Comment:
```suggestion
Using a `NODE` or `RELATIONSHIP` instead of a `MAP` on the RHS of a xref:clauses/set.adoc[`SET`] properties clause is no longer supported.
Instead, use the xref:functions/scalar.adoc#functions-properties[`properties()`] function to get the map of properties from nodes or relationships, which can then be used in the `SET` clause.
```